frontend/src/pages/visualizeOptionsPage.js

```diff
@@ -113,11 +143,11 @@ export function VisualizeOptionsPage(props) {
                             WEIGHT
                         </Text>
                         <Text fw={600} size='md' mt='5em'> Lens Only</Text>
-                        <Text fw={400} size='md' mt='.5em'> {weight.lens}g</Text>
+                        <Text fw={400} size='md' mt='.5em'> {lensWeight}g</Text>
                         <Text fw={600} size='md' mt='3em'> Frames Only</Text>
-                        <Text fw={400} size='md' mt='.5em'> {weight.frame}g</Text>
+                        <Text fw={400} size='md' mt='.5em'> {frameWeights[frameName]}g</Text>
```

Comment:
Imo, it would be nice for this to display 2 significant figures like `12.00g` just for consistency.

Reply:
sure